spp_manual_eligibility/wizard/create_program_wizard.py

```diff
@@ -0,0 +1,46 @@
+# Part of OpenG2P. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import fields, models
+
+# from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class G2PCreateNewProgramWiz(models.TransientModel):
+    _inherit = "g2p.program.create.wizard"
+
+    eligibility_kind = fields.Selection(selection_add=[("manual_eligibility", "Manual Eligibility")])
+
+    def _get_default_eligibility_manager_val(self, program_id):
```

Comment:
use `super` here so that you will not overwrite other modules that is extending this method

kindly refer to the following files
- spp_import_dci_api/wizard/create_program_wizard.py
- spp_exclusion_filter/wizard/create_program_wizard.py

Reply:
Done